tdrs-backend/tdpservice/users/permissions.py

```diff
@@ -51,6 +51,19 @@ def has_permission(self, request, view):
         return is_in_group(request.user, "Data Prepper")
 
 
+class CanDownloadReport(permissions.BasePermission):
+    """Permission for report download."""
+
+    def has_permission(self, request, view):
+        """Check if a user can download file."""
+        if is_in_group(request.user, "OFA Admin") and view.kwargs.get("stt"):
+            return True
+        elif request.user.is_authenticated:
+            return True
+        else:
+            return False
```

Comment:
```suggestion
    def has_permission(self, request, view):
        """Check if a user can download file."""
        is_ofa_admin = bool(
            is_in_group(request.user, 'OFA Admin') and view.kwargs.get('stt')
        )
        return is_ofa_admin or is_own_stt(request, view)
```
We shouldn't allow any authenticated user to download reports / metadata about submitted reports.